app/models/heartbeat.rb

```diff
@@ -19,7 +19,7 @@ def services_up?
 
   def memcached_up?
     host = ENV["MEMCACHE_SERVERS"]
-    memcached_client = Dalli::Client.new("#{host}:11211")
+    memcached_client = Dalli::Client.new(ENV["MEMCACHE_SERVERS"])
```

Comment:
This is the same except that it no longer specifies the port?

Reply:
the result is exactly the same. i'm following the convention used in lupo which uses the environment variable. the environment variable does have the port.

Reply:
Could get rid of line 21